flash_examples/predict/image_embedder.py

```diff
@@ -0,0 +1,28 @@
+import torch
+
+from flash.core.data import download_data
+from flash.vision import ImageEmbedder
+
+if __name__ == "__main__":
+
```

Comment:
```suggestion
def main():
```

Reply:
I prefer to keep ` __name__ == "__main__"`